tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -1,122 +1,161 @@
 ---
 title: Console Audit Logging
-summary: Learn about the log auditing feature for the TiDB Cloud console.
+summary: Learn about the audit logging feature for the TiDB Cloud console.
 ---
 
 # Console Audit Logging
 
-TiDB Cloud provides the audit logging feature for TiDB Cloud console operations, which records the history of user access details (such as user login to the Console and cluster creation operations).
+TiDB Cloud provides the console audit logging feature to help you track various behaviors and operations of users on the [TiDB Cloud console](https://tidbcloud.com). For example, you can track operations, such as inviting a user to join the organization and creating a cluster.
 
-> **Note:**
->
-> Currently, the **audit logging** feature is experimental. The output is subject to change.
+## Prerequisites
 
-## Limitations
+- You must be the owner or Audit Admin of your organization in TiDB Cloud. Otherwise, you cannot see the console audit logging-related options in the TiDB Cloud console. For more information, see [Manage role access](/tidb-cloud/manage-user-access.md#manage-role-access).
```

Comment:
```suggestion
- You must be in the Owner or Audit Admin role of your organization in TiDB Cloud. Otherwise, you cannot see the console audit logging-related options in the TiDB Cloud console. For more information, see [Manage role access](/tidb-cloud/manage-user-access.md#manage-role-access).
```